templates/bot/ts/command-and-response/package.json

```diff
@@ -8,7 +8,7 @@
     "scripts": {
         "dev:teamsfx": "env-cmd --silent -f .env.teamsfx.local npm run dev",
         "dev": "nodemon --watch ./src --exec node --inspect=9239 --signal SIGINT -r ts-node/register ./src/index.ts",
-        "build": "tsc --build",
+        "build": "tsc --build && shx cp -r ./src/adaptiveCards ./lib/src",
```

Comment:
Does this conflict with #4962 ?

Reply:
Ah~yeah, then @KennethBWSong will need to merge with this change :)

Reply:
Already include in pr #4962 , thx!